readthedocs/projects/models.py

```diff
@@ -731,9 +731,9 @@ def alias(self):
         if self.is_subproject:
             return self.superprojects.first().alias
 
-    def subdomain(self):
+    def subdomain(self, use_custom_domain=True):
```

Comment:
Do we actually want `use_canonical_domain` here? Seems like that is the actual case where the domain would be changed, and is clearer in cases of multiple custom domains?